src/content/contributing/translation-program/translation-versions/index.md

```diff
@@ -0,0 +1,131 @@
+- [Intro to dapps](/developers/docs/dapps)
+- [Web2 vs Web3](/developers/docs/web2-vs-web3/)
+- [Accounts](/developers/docs/accounts/)
+- [Transactions](/developers/docs/transactions/)
+- [Blocks](/developers/docs/blocks/)
+- [Ethereum virtual machine (EVM)](/developers/docs/evm/)
+- [Gas](/developers/docs/gas/)
+- [Nodes and clients](/developers/docs/nodes-and-clients/)
+- [Nodes as a service](/developers/docs/nodes-and-clients/nodes-as-a-service/)
+- [Networks](/developers/docs/networks/)
+- [Consensus mechanisms](/developers/docs/consensus-mechanisms/)
+- [Proof of work](/developers/docs/consensus-mechanisms/pow/)
+- [Mining](/developers/docs/consensus-mechanisms/pow/mining/)
+- [Proof of stake](/developers/docs/consensus-mechanisms/pos/)
+- [Introduction to the Ethereum stack](/developers/docs/ethereum-stack/)
+- [Smart contracts](/developers/docs/smart-contracts/)
+- [Smart contract languages](/developers/docs/smart-contracts/languages/)
+- [Smart contract anatomy](/developers/docs/smart-contracts/anatomy/)
+- [Smart contract libraries](/developers/docs/smart-contracts/libraries)
+- [Testing smart contracts](/developers/docs/smart-contracts/testing)
+- [Compiling smart contracts](/developers/docs/smart-contracts/compiling)
+- [Deploying smart contracts](/developers/docs/smart-contracts/deploying)
+- [Composability](/developers/docs/smart-contracts/composability/)
+- [Deployment networks](/developers/docs/development-networks/)
+- [Development frameworks](/developers/docs/frameworks/)
+- [JavaScript APIs](/developers/docs/apis/javascript/)
+- [Backend APIs](/developers/docs/apis/backend/)
+- [Data and analytics](/developers/docs/data-and-analytics/)
+- [Block explorers](/developers/docs/data-and-analytics/block-explorers/)
+- [Security](/developers/docs/security/)
+- [Storage](/developers/docs/storage/)
+- [Integrated Development Environments (IDEs)](/developers/docs/ides/)
+- [Programming languages](/developers/docs/programming-languages/)
+- [Delphi](/developers/docs/programming-languages/delphi/)
+- [.NET](/developers/docs/programming-languages/dot-net/)
+- [Golang](/developers/docs/programming-languages/golang/)
+- [Java](/developers/docs/programming-languages/java/)
+- [JavaScript](/developers/docs/programming-languages/javascript/)
+- [Python](/developers/docs/programming-languages/python/)
+- [Rust](/developers/docs/programming-languages/rust/)
+- [Standards](/developers/docs/standards/)
+- [Token standards](/developers/docs/standards/tokens/)
+- [ERC-20](/developers/docs/standards/tokens/erc-20/)
+- [ERC-721](/developers/docs/standards/tokens/erc-721/)
+- [Orcles](/developers/docs/oracles/)
+- [Scaling](/developers/docs/scaling/)
+- [Layer 2 Rollups](/developers/docs/scaling/layer-2-rollups/)
+- [State channels](/developers/docs/scaling/state-channels)
+- [Sidechains](/developers/docs/scaling/sidechains)
+- [Plasma](/developers/docs/scaling/plasma)
+- [Validium](/developers/docs/scaling/validium/)
```

Comment:
I'm fine with it either way, but any thoughts on simplifying this to just everything in "/developers/docs" is v2.1?